models/Program.ts

```diff
@@ -0,0 +1,31 @@
+// TODO: weekday field got moved to class so we can move this to Class.ts after
+// export enum weekdays {
+//     MON = "MON",
+//     TUE = "TUE",
+//     WED = "WED",
+//     THU = "THU",
+//     FRI = "FRI",
+//     SAT = "SAT",
+//     SUN = "SUN",
+// }
+
+/**
+ * Enum for the program format
+ */
+export enum programFormat {
+    online = "online",
```

Comment:
do you need to specify the strings?

Reply:
I think we might need it `inPerson = "in-person"` since the schema has it in snake case right now. After #26 is merged, I think we can remove